float64 pos_x # x position of the robot 
float64 pos_y # y position of the robot 
float64 vel_x # x velocity of the robot
float64 vel_ang_z # angular velocity of the robot